src/web/components/Core/Loading.stories.tsx

```diff
@@ -0,0 +1,17 @@
+import type { Meta, StoryObj } from '@storybook/react';
+
+import { Loading } from './Loading';
+
+const meta: Meta<typeof Loading> = {
```

Comment:
Let's add a `title` so that it shows up with the rest of the shared components:
```
const meta: Meta<typeof Loading> = {
  title: 'Shared Components/Loading',
  component: Loading,
};
```